readme.md

```diff
@@ -0,0 +1,265 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="/murple_logo.png" alt="logo" width="200"  height="auto" />
```

Comment:
- [ ] As a general requirement you should remove the template logo. This would make your repo looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)
Optionally, you can replace it with your own logo if you have one :+1:.